swhkd/src/daemon.rs

```diff
@@ -650,6 +650,11 @@ pub async fn send_command(
 fn get_uid() -> Result<u32, Box<dyn Error>> {
     let status_content = fs::read_to_string(format!("/proc/{}/loginuid", std::process::id()))?;
     let uid = status_content.trim().parse::<u32>()?;
+    if uid == u32::MAX {
+        return Err("got invalid loginuid value 4294967295 ('-1' in unsigned long), meaning loginuid is unset. \
```

Comment:
Why are we hardcoding u32::MAX in the error log? It's different machine to machine.